dirty_cat/datasets/fetching.py

```diff
@@ -166,6 +166,71 @@ def fetch_openml_dataset(
     }
 
 
+def fetch_world_bank_data(dataset_id: str, indicator: str,
+                          data_directory: Path = get_data_dir(),
+                          ) -> Dict[str, Any]:
```

Comment:
This should be a private function, we don't want users to use it directly

```suggestion
def _fetch_world_bank_data(dataset_id: str, indicator: str,
                           data_directory: Path = get_data_dir(),
                           ) -> Dict[str, Any]:
```

(watch out for references in the code, they need to be modified as well)